apps/src/StudioApp.js

```diff
@@ -3,6 +3,7 @@
 
 var parseXmlElement = require('./xml').parseElement;
 var utils = require('./utils');
+var dropletUtils = require('./dropletUtils');
```

Comment:
This looks like it will increase the code size for non-droplet levels. Do you foresee a path for loading dropletUtils only for editCode levels, or are we saying dropletUtils is small enough that it's not worth it?